README.md

```diff
@@ -0,0 +1,163 @@
+• ventilation_mask_idh: Mask describing if ventilation is activated at hour idh of the simulation,
+• occupancy_idh: Occupancy profile at hour idh of simulation,
+• TAMB_idh: Outside temperatures at hour idh of simulation,
+More information on the following weather input data (obtained hourly) can be found at [ammonit.com](https://www.ammonit.com/en/wind-solar-wissen/solarmessung)
+
+• DNI_idh: Direct Normal Irradiance at hour idh,
+• RHUM_idh: Humidity at hour idh,
+• IBEAM_H_idh: Direct Horizontal Irradiance at hour idh,
+• IBEAM_N_idh: Direct Normal Irradiance at hour idh,
+• IDIFF_H_idh: Diffuse Horizontal Irradiation at hour idh,
+• IGLOB_H_idh: Global Horizontal Irradiance at hour idh,
+Output files contain the times series to be predicted hourly from the input. They comprise 5377 columns per sample: the first index column contains the unique sample identifiers (corresponding to the input identifiers) while the other 5376 columns contain the inside temperature of the building and various consumptions in kW/hour, normalized by a factor 2e4 for a better numerical interpretability of the loss:
+
+• T_INT_OFFICE_idh: Inside temperatures,
+• Q_AC_OFFICE_idh, Q_HEAT_OFFICE_idh, Q_AHU_C_idh, Q_AHU_H_idh: Cooling, heating and Air handling unit (AHU which can cool or heat outside air) consumptions,
+• Q_EQP_idh, Q_LIGHT_idh: PC and Lights consumptions,
+• Q_PEOPLE_idh: "Free" heating power due to human activities in the building,
+The solution files submitted by participants shall follow this output dataset format (i.e contain the above 5377 columns, where the index values correspond to the input test data). An example submission file containing random predictions is provided.
+
+7500 samples (i.e. lines) are available for the training datasets while 500 observations are used for the test datasets.
+
+### Benchmark Description
+
+Because of the sequential nature of the data, we chose a recurrent neural network as a benchmark. Data: We began by creating our input vectors using, at each time step, both the time series values and the parameters of the building. This amount to an input tensor of shape (672, 37). The input and output values are then normalized to improve convergence of the network. Network: We deployed a simple network in PyTorch, stacking 3 LSTM layers feeding into a fully connected layer. No dropout or batch normalization were applied. Training: We trained the network on the dataset using the mean squared error loss, and the Adam optimizer.
+
+---
+
+## Requirements
+
+- [Install dependencies](#installation)
+- [Register and login to challenge](#oze-challenge-benchmark)
+- [Configure `.env.test.local`](#dot-env-environment-variables)
+
+### Installation
+
+Clone repository with the following command in the terminal
+
+```terminal
+git clone https://github.com/maxjcohen/ozechallenge_benchmark.git
+cd ozechallenge_benchmark
+virtualenv -p python3 .env
+pip install -r requirements
```

Comment:
Here we should add a line to activate the environment, I had issue with some developers not being used to activate it once created.
```bash
virtualenv -p python3 .env
source .env/bin/activate
pip install -r requirements
```

Reply:
Alright.